### PR TITLE
updating docs

### PR DESCRIPTION
**Pull request by Joe-Cun-Civica**
updating documentation to better reflect recent updates

### CODE REVIEW

File: docs/contribution-guidelines/the-repository-structure.md [hunk 1]
```diff
@@ -24,14 +24,14 @@
 
 - `wdio.conf.js`
```
Comment (kieran-clifton-civica) [inline docs/contribution-guidelines/the-repository-structure.md]: should be wdio.conf.cjs

File: docs/contribution-guidelines/writing-a-new-feature-file.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- Avoid excessive setup steps if a QA route can inject data and jump directly to the page under test.
+- Prefer happy-path scenarios, then add edge cases as separate scenarios.
+- Use real user language, not implementation details.
+
+## Running the new feature
+
+To run the new feature file locally:
+
+```bash
+npx wdio run wdio.conf.mjs --spec src/features/<your-feature-file>.feature
```
Comment (kieran-clifton-civica) [inline docs/contribution-guidelines/writing-a-new-feature-file.md]: should be wdio.conf.cjs

File: docs/contribution-guidelines/writing-a-new-feature-file.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    When I click on the element "customerType.endConsumer"
+    Then the next page should contain the text "Your details"
+```
+
+## Step definitions
+
+- Feature files call step definitions from `src/steps/given.js`.
+- Reuse existing steps wherever possible.
+- Keep step text readable and aligned with the existing style.
+- If you need a new step, add it to `src/steps/given.js` and keep it generic enough to be reused by future scenarios.
```
Comment (kieran-clifton-civica) [inline docs/contribution-guidelines/writing-a-new-feature-file.md]: I think there needs to be mention of all step files 

```suggestion
## Step definitions

- Feature files use steps defined across three files in `src/steps/`:
  - `given.js` - `Given` steps (setup and navigation)
  - `when.js` - `When` steps (actions and interactions)
  - `then.js` - `Then` steps (assertions and checks)
- Reuse existing steps wherever possible.
- Keep step text readable and aligned with the existing style.
- If you need a new step, add it to the file that matches its keyword (`Given`, `When`, or `Then`) and keep it generic enough to be reused by future scenarios.
```

File: docs/contribution-guidelines/writing-a-new-step-definition.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  myAction
+);
+```
+
+4. Implement the action in `src/support/action/`.
+5. Keep step text reusable and avoid page-specific details.
+6. Reuse existing support actions when possible.
+7. If the step needs a new selector, add it to `src/pageObjects`.
+
+Run the new step by executing the relevant feature file or suite.
```
Comment (kieran-clifton-civica) [inline docs/contribution-guidelines/writing-a-new-step-definition.md]: again this only mentions the given step 


```suggestion
# Writing a new step definition

1. Open the step file that matches your keyword:
   - `src/steps/given.js` for `Given` steps
   - `src/steps/when.js` for `When` steps
   - `src/steps/then.js` for `Then` steps

2. Add a new entry with a regex and a handler.

Example (`When` step):

```js
import myAction from "../support/action/myAction.js";

When(
  /^I do something custom "([^"]*)"$/,
  { wrapperOptions: { retry: 2 } },
  myAction
);
Implement the handler in the correct support directory:
src/support/action/ for Given and When steps (actions)
src/support/check/ for Then steps (assertions)
Keep step text reusable and avoid page-specific details.
Reuse existing support functions when possible.
If the step needs a new selector, add it to src/pageObjects.
Run the new step by executing the relevant feature file or suite.
```

**Review by kieran-clifton-civica — COMMENTED** [review]
small changes then its all good

**Review by kieran-clifton-civica — APPROVED** [review]

**Review by ernestas-leliuga — APPROVED** [review]
